plutus-ledger-api/src/PlutusLedgerApi/V1/Bytes.hs

```diff
@@ -1,7 +1,9 @@
--- editorconfig-checker-disable-file
-{-# LANGUAGE DeriveAnyClass  #-}
-{-# LANGUAGE DerivingVia     #-}
-{-# LANGUAGE TemplateHaskell #-}
+{-# LANGUAGE BlockArguments    #-}
+{-# LANGUAGE DeriveAnyClass    #-}
+{-# LANGUAGE DerivingVia       #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE TemplateHaskell   #-}
+{-# LANGUAGE TypeApplications  #-}
 
 {-# OPTIONS_GHC -Wno-orphans            #-}
```

Comment:
There are a bunch of these in plutus-ledger-api. What are they for?

Reply:
Not sure. Pretty likely they became obsolete. I've removed them, lets  and if build doesn't fail (which it didn't locally).